ansible/MANUAL_STEPS.md

```diff
@@ -1,4 +1,17 @@
-# Manual steps required to run ansible on machines
+# Manual steps required to setup machines
+
+## `release-*` machines
+
+Once setup, they must have `~iojs/.ssh` cloned from another machine, so they
+have the ssh setup and keys required to upload release artifacts to the
+nodejs.org web server.
```

Comment:
Probably should list exactly what is needed: (1) a `config` containing a `Host node-www` and (2) a `node-www` private key.